std/concurrency.d

```diff
@@ -2502,14 +2502,20 @@ auto ref initOnce(alias var)(lazy typeof(var) init, Mutex mutex)
     return var;
 }
 
+deprecated("use the overload that takes a `shared(Mutex)`")
+auto ref initOnce(alias var)(lazy typeof(var) init, Mutex mutex)
```

Comment:
Not sure about this. Should we deprecate code that's been using `__gshared` mutexes without issue?

Reply:
I've un-deprecated the unshared overload. Not adding a comment about future deprecation, because it's not clear (to me) what's going to happen.